app/components/search_result_caption_component.html.erb

```diff
@@ -0,0 +1,7 @@
+  <span class="govuk-!-margin-right-4">
+    <% if searched.present? %>
+      <strong><%= count %></strong> <%= item_label %> found matching ‘<%= searched %>’ of <strong><%= total_count %></strong> total <%= item %>. <%= govuk_link_to("Clear search", request.path) %>
+    <% else %>
+      <strong><%= count %></strong> total <%= item %>.
+    <% end %>
```

Comment:
Looks like the `title` is missing here

Reply:
Title has been replaced with item but I can change that for clarity if needed?